_posts/2023-09-26-Hacktoberfest-2023.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Hacktoberfest 2023"
+date:   2023-09-26
+tags:   Hacktoberfest
+author: rchakrab
+description: Article about Hacktoberfest participation and small descriptions about the participating projects and how to contribute in it.
+---
+
+= Hacktoberfest 2023
+
+image::hacktoberfest/hacktoberfest2023.png[Content Browser]
+
+https://hacktoberfest.com/[Hacktoberfest] is back, and it's time to channel your inner Open Source superhero! This annual event unites developers of all levels, from seasoned pros to enthusiastic beginners, in a shared mission to contribute to Open Source projects and celebrate the spirit of collaboration in software development. Join us for this month-long celebration of contributions as we work together, one pull request at a time.
+
+== What is Hacktoberfest?
+
+Hacktoberfest is a month-long event in October that encourages Open Source contributions and developers worldwide to give back to the global tech community. It's a fantastic opportunity for the contributors to showcase their skills and get to know about other Open Source communities.
+
+To participate you simply need to:
+
+
+* *Register:* https://hacktoberfest.com/auth/[Sign up] on the official Hacktoberfest website to get started.
+* *Discover Projects:* Explore the Hacktoberfest website to find a project that sparks your interest. You can also check out the curated list of projects we've prepared for you.
+* *Follow the Project guidelines:* Read the respective Contributing Guide, Code of Conduct, and follow best practices like forking the repository, creating a new branch, and making a meaningful contribution.
+* *Contribute:* Select an issue that matches your interest and start working on it. You'll have the option to receive a Hacktoberfest t-shirt or have a tree planted in your name as a token of appreciation.
+
+== Get started with our projects
+
+This list contains some really interesting projects that are under the WildFly umbrella, waiting for your magic touch. There’s something for everyone from Java and Ansible to TypeScript and JavaScript.
+
+. *WildFly:* WildFly is a powerful, modular, and lightweight application server that empowers you to build amazing applications.
+* Here is the https://github.com/wildfly/wildfly[GitHub repository], https://github.com/wildfly/wildfly/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/WFLY-17539?filter=12421535[Hacktoberfest Issues link].
+
+. *WildFly Elytron:* WildFly Elytron is a new WildFly sub-project that is completely replacing the combination of PicketBox and JAAS as the WildFly client and server security mechanism.
```

Comment:
This needs to be updated, as Elytron is not new now and the fact they replaced Picketbox is no longer a key thing to mention.

Reply:
@Skyllarr can you please provide me a description for WildFly Elytron?

Reply:
@RanabirChakraborty WildFly Elytron is a set of APIs and SPIs for application server security and client side security. Within the WildFly server, WildFly Elytron is used to secure applications deployed to the server as well as management access to the server. It is a standalone library that can also be used in other Java server environments.